README.md

```diff
@@ -920,6 +920,9 @@ arXiv, 2023.
 - <a name="Liu23tpami"></a>Liu, S., Dragotti, P.L.,    
 *[Sensing Diversity and Sparsity Models for Event Generation and Video Reconstruction from Events](https://doi.org/10.1109/TPAMI.2023.3278940)*,    
 IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2023. [Code](https://github.com/pld-group/V2E2V)
+- <a name="Qu24aaai"></a>Qu, Q., Shen, Y., Chen, X., Chung, Y.Y., Liu, T.,    
+*[E2HQV: High-Quality Video Generation from Event Camera via Theory-Inspired Model-Aided Deep Learning](https://arxiv.org/abs/2401.08117)*,    
```

Comment:
Is there an official AAAI link already?
If not, we can keep the arXiv one. 
When the official AAAI link (DOI) is available we can update the link in the title and add and arXiv link in [PDF] next to [Code]. Thanks

Reply:
The official AAAI link is not available at the moment. We will continue using the arXiv link for now. As soon as the official link becomes available, I will promptly update the link in the title, and move arXiv link to [PDF].